### DISCUSSION TITLE
Where to download the lastest compiled firmware?

### DISCUSSION THREAD

**Discussion by netweaver1970** [vortigont/Songguo-PTS200#8] [Q&A]
Hi,
I can see the Firmware was built by the PlatformIO CI, including the PowerRamp PR but I can't seem to find the resulting downloadable (and installable) file. Can you point me to it? Am I being blind? Or just no aware where to look?
My Russian is not great, otherwise I would have used Discord...

Thanks.

**Comment by vortigont** [comment]
Hi!
I do not release precompiled binaries (yet). It is still under development. You have to build it yourself from the master branch.
Let me know if you have problems with that then I might come up with some CI workflow that release binary artefacts.
Though you need to flash it via PIO or espflashtool anyway. MSC update from factory firmware I've never tested.

**Comment by netweaver1970** [comment]
Ok, it all went smoothly, I used git clone into the PIO projects folders, did the firmware install to the soldering iron from PIO. All very smooth. Heating up (after enabling Ramping mode in the menu) went flawless on 2 PD wall chargers, one simple KeKe-PD001 20W charger was nicely delivering up to 20W on 12V. Another one, a beefy 140W Baseus PD charger was delivering up to +70W, at 20V. Maybe it would draw more current if need be, will test more later. The auto heat-up after idle when pickig up the iron doesn't seem to work. But a quick single middle press woke it up fine. So far already very useful, will keep testing with other PD chargers when I have them available. Thanks.

**Comment by vortigont** [comment]
thanks for the feedback, @netweaver1970, appreciate it!
After idle time in standby more it won't auto heat-up on motion, this is intended behavior. Do not want the iron to get heated on some random push and shove.
Currently a bit busy with other tasks, but have more ideas to implement in my mind.
